src/main/java/RoleyPoley.java

```diff
@@ -0,0 +1,92 @@
+import java.util.Scanner;
+import java.util.List;
+import java.util.ArrayList;
+
+public class RoleyPoley {
+    public static void main(String[] args) {
+        Task[] taskList = new Task[100];
+        greet();
+        echo(taskList);
+    }
+
+    public static void displayList(Task[] taskList) {
+        for ( int i = 1; i < taskList.length; i++) {
+            if (taskList[i] == null) {
+                break;
+            }
+            System.out.println("\t" + i + ".[" + taskList[i].getStatusIcon() + "]" + taskList[i].description);
+        }
+    }
+
+    public static void createLine() {
```

Comment:
Maybe can just call print a single time for a string of "-------------------" instead of using a for loop to repeatedly call print.